### ISSUE TITLE
Landscape UI - Unable to close FAB menu and notch is blocking some content from the preferences page - Reported by: @kakajann 

### ISSUE THREAD

**Issue by isagoico** [Expensify/App#5926]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Navigate to LHN 
2. Open FAB menu
3. Change orientation to landscape
4. Change back orientation to portrait
5. Navigate to settings
6. Open preferences
7. Change orientation to landscape. 

## Expected Result:
UI should not break after changing orientation to landscape. 

## Actual Result:
User is unable to exit the FAB menu in preferences some UI elements are blocked by notch.
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platform:

Where is this issue occurring?

- iOS
- Android

**Version Number:** 1.1.8-0

**Reproducible in staging?:** Yes 
**Reproducible in production?:** Yes 

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation
![image](https://user-images.githubusercontent.com/44479856/137759340-0ed407d5-c137-46b5-83bb-865cc1abef9a.png)
![image](https://user-images.githubusercontent.com/44479856/137759420-c131121b-dc13-41b5-aada-157b049d768d.png)

**Upwork job link:** https://www.upwork.com/jobs/~017190bb5038fac386

**Issue reported by:** @kakajann 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1634299085343500

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
Triggered auto assignment to @pecanoro (`Engineering`), see https://stackoverflow.com/c/expensify/questions/4319 for more details.

**Comment by shawnborton** [comment]
I think this is my favorite option so far:
![image](https://user-images.githubusercontent.com/2319350/139720541-40bd1a61-96c0-4712-98a1-08b49b24beff.png)

**Comment by kakajann** [comment]
Any news?

 I can start working if you like my proposal

**Comment by sidferreira** [comment]
@kakajann in case you are chosen, keep in mind they have `react-native-safe-area-context` available

**Comment by michaelhaxhiu** [comment]
Next step is to wait for @Julesssss to confirm the best solution proposal (in order of first come first serve).

**Comment by michaelhaxhiu** [comment]
Note: Jules will likely get to this issue on Monday. Thanks for your patience.

**Comment by mallenexpensify** [comment]
We are placing all non-critical issues on hold while we're on a company offsite.  The hold is expected to be lifted on 11/19 (but could go longer).  For any PRs that are submitted during the hold, we will add a $250 bonus payment.

**Comment by Julesssss** [comment]
To be honest, I prefer the solution which adds additional padding, rather than letterboxing the page. But this is really a design issue, so I delegate to Shawn here.  

Lets hire @kakajann for [this solution](https://github.com/Expensify/App/issues/5926#issuecomment-946335986).

**Comment by mallenexpensify** [comment]
@kakajann can you apply to the issue and comment here once you have?
https://www.upwork.com/jobs/~017190bb5038fac386

**Comment by kakajann** [comment]
Just applied. I'll raise a PR in a few days

**Comment by mallenexpensify** [comment]
Hired @kakajann in Upwork. 
Reminder to self... @kakajann also reported the issue so the $250 reporting bonus is applicable

**Comment by Julesssss** [comment]
Merged, awaiting payment

**Comment by michaelhaxhiu** [comment]
Bug reporting bonus = $250
Job payout = $250
Company Offsite Hold = $250

Total = $750.

Just paid it, we should be all good here.

**Comment by botify** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.1.16-10 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/6291


If no regressions arise, payment will be issued on 2021-12-06. :confetti_ball:

**Comment by parasharrajat** [comment]
There is a regression from this issues' PR. https://github.com/Expensify/App/issues/6530. cc: @kakajann

**Comment by Julesssss** [comment]
The early payment is my fault, I mistakenly started the 7-day countdown when the issue was merged, instead of waiting for it to be deployed. 

I agree this should still be treated as a regression that should be fixed 👍

**Comment by parasharrajat** [comment]
Thanks, @kakajann . Please request a review from me when you do and tag the other issue as well.

**Comment by botify** [comment]
The solution for this issue has been :rocket: deployed to production :rocket: in version 1.1.18-3 and is now subject to a 7-day regression period :calendar:. Here is the list of pull requests that resolve this issue:

- https://github.com/Expensify/App/pull/6559


If no regressions arise, payment will be issued on 2021-12-15. :confetti_ball:

**Comment by michaelhaxhiu** [comment]
Is this good to close now?

**Comment by mallenexpensify** [comment]
Looks like it, reopen if needed
